internal/clients/bbnclient/types.go

```diff
@@ -50,6 +51,7 @@ func FromBbnStakingParams(params stakingtypes.Params) *StakingParams {
 		DelegationCreationBaseGasFee: params.DelegationCreationBaseGasFee,
 		AllowListExpirationHeight:    params.AllowListExpirationHeight,
 		BtcActivationHeight:          params.BtcActivationHeight,
+		MaxFinalityProviders:         params.MaxFinalityProviders,
```

Comment:
Actually now I think just adding this field might be not sufficient - they can update staking params, version will be incremented for old babylon-indexer (with struct without this field) and we won't overide it later (when new version of indexer deployed)

Reply:
I wanted to put the script under `babylon-staking-indexer/cli/phase-3` directory, but I needed access to existing functions in `babylon-staking-indexer/cli`and there was cycle dependency - that's why new script under `cli` dir